apps/profile-web/src/components/QuickAccess/index.tsx

```diff
@@ -1,14 +1,64 @@
 import React from "react";
 
-// import styled, { AnyStyledComponent } from "styled-components";
+import styled, { AnyStyledComponent } from "styled-components";
 
-// const Temp: AnyStyledComponent = styled.div`
-// `;
+const QAWrapper: AnyStyledComponent = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 280px;
+  height: 174px;
+  border-radius: 12px;
+  background: #113366;
+`;
+
+const QAHeader: AnyStyledComponent = styled.div`
+  padding-top: 13px;
+  padding-left: 20px;
+  padding-bottom: 25px;
+  font-weight: 800;
+  font-size: 1.25rem;
+  color: #FFFFFF;
+  width: 100%;
+`;
+
+const ItemWrapper: AnyStyledComponent = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+const QAItem: AnyStyledComponent = styled.a`
+  padding-bottom: 10px;
+  padding-left: 20px; 
+  color: #FFFFFF;
+  font-weight: 600;
+  font-size: 1.0625rem;
+  max-width: fit-content;
+`;
 
 export const QuickAccess: React.FC<{}> = () => {
   return (
-    <div>
-      <h1>Quick Access</h1>
-    </div>
+    <QAWrapper>
+      <QAHeader>Quick Access</QAHeader>
+      <ItemWrapper>
+        <QAItem href = "https://google.com">
```

Comment:
```suggestion
        <QAItem href="https://google.com">
```
Run Prettier so you don't have extra white space.